arches_lingo/src/arches_lingo/components/user/UserInteraction.vue

```diff
@@ -52,13 +52,11 @@ const greeting = computed(() => {
 </script>
 
 <template>
-    <div style="display: flex; align-items: center">
+    <div style="display: flex; align-items: center; gap: var(--p-content-gap)">
         <span v-if="user">{{ greeting }}</span>
         <Button
-            style="margin-left: 1rem"
+            :label="$gettext('Sign out')"
```

Comment:
Without using the label prop, it wasn't using the tokens for button label font weight (not noticeable until I started overriding that weight in core arches)